build.sh

```diff
@@ -99,12 +101,16 @@ function pull_image {
       echo "The image $image_name is already pulled."
       continue
     fi
+
     # Try pulling the image to see if it is accessible
-    if ! docker pull "$image_name"; then
-      echo "Pulling image $image_name failed. Let's wait 2 seconds and try one more time."
-      sleep 2
-      docker pull "$image_name"
-    fi
+    # WORKAROUND: Since Fedora registry sometimes fails randomly, let's try it more times
+    while ! docker pull "$image_name"; do
+      ((loop++))
+      echo "Pulling image $image_name failed."
+      [ "$loop" -gt "$loops" ] && echo "It happened $loops times. Giving up." && return 1
```

Comment:
Just nit pick what about:
```suggestion
      [ "$loop" -gt "$loops" ] && { echo "It happened $loops times. Giving up." ; return 1; }
```

Reply:
Thanks